lib/open_project.rb

```diff
@@ -44,4 +44,11 @@ module OpenProject
   def self.logger
     ::OpenProject::Logging::LogDelegator
   end
+
+  def self.httpx
+    HTTPX
+      .plugin(:persistent)
+      .plugin(:basic_auth)
+      .plugin(:webdav)
+  end
```

Comment:
🟢 Nice unification 👍🏾  One thing that might change in the future is that not all plugins will be needed in all contexts- and I wonder if there's expense in eager loading them.  But as they say, _"we can cross that bridge when when we get there"_ 😄

Reply:
The plugins, in general, just load a couple of files and add a couple of hooks to the session, so shouldn't be an issue.

Reply:
Yes, it should not be an issue and plugins code is loaded once we create a session like that for the first time.